.github/CODEOWNERS

```diff
@@ -6,6 +6,9 @@
 /Tests/Marketplace/core_packs_list.json @yaakovpraisler @bakatzir @GuyLibermanPA @demisto/content-leaders
 /Tests/Marketplace/core_packs_mpv2_list.json @yaakovpraisler @bakatzir @GuyLibermanPA @demisto/content-leaders
 
+# native image
```

Comment:
Can you give more context?
I guess it's too late to rename the config file as well?
To something like: `/Tests/docker_native_image_config.json`
```suggestion
# Docker native image
```

Reply:
its not late, i can still do it, ill open another PR